src/AerospikeCache.php

```diff
@@ -41,8 +41,9 @@ protected function doFetch(array $ids): array
 
         foreach ($records as $record) {
             if ($record['metadata'] !== null) {
-                $result[$record['key']['key']] = isset($record['bins'][self::WRAPPER_NAME]) ?
-                    $record['bins'][self::WRAPPER_NAME] : null;
+                $result[$record['key']['key']] = isset($record['bins'][self::WRAPPER_NAME])
```

Comment:
it could be just `$record['bins'][self::WRAPPER_NAME] ?? null`

Reply:
ok changed